pkg/engine/testdata/unsupported_edge.err.json

```diff
@@ -6,21 +6,5 @@
       "Source": "aws:cloudfront_distribution:cloudfront_distribution_2",
       "Target": "aws:rds_instance:mydb"
     }
-  },
-  {
-    "classification": "network",
-    "error_code": "edge_unsupported",
-    "satisfaction_edge": {
-      "Source": "aws:cloudfront_distribution:cloudfront_distribution_2",
-      "Target": "aws:rds_instance:mydb"
-    }
-  },
```

Comment:
This change seems unintentional - this should still fail, right? Even if it should no longer fail, this test is supposed to cover the unsupported edge, so could you update the input if that's the case?

Reply:
nope its intentionaly because we see that theres a path now that we can go cloudfront -> LB - > RDS somehow i guess. It still does fail on one of the other paths of classifications though